docs/source/api_reference.rst

```diff
@@ -1,3 +1,4 @@
+=============
 API Reference
 =============
```

Comment:
can we add the different label time transforms, plotting methods, etc to the api reference?

Reply:
Added tables to API reference in [this](https://github.com/FeatureLabs/compose-ml/pull/29/commits/6cb5017ff13fe2d9d8814512caa50ceabb4c9cf4#diff-18294892d540ca2ec3e466841181ff63R27) commit.